djangoplicity/mailinglists/migrations/0001_initial.py

```diff
@@ -54,7 +54,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('on_event', models.CharField(max_length=50, db_index=True)),
-                ('action', models.ForeignKey(to='actions.Action')),
+                ('action', models.ForeignKey(to='actions.Action', on_delete=models.CASCADE)),
```

Comment:
In migrations, the **on_delete** is assigned differently.

Follow this: 
![image](https://user-images.githubusercontent.com/36459449/97223817-4849cd80-179e-11eb-9704-6f63187a083d.png)

Reply:
Import this before:

```python
import django.db.models.deletion
```